### PR TITLE
always load selected value in autocomplete

### PR DESCRIPTION
**Pull request by tomaskikutis**
SDBELGA-552

### CODE REVIEW

File: scripts/core/ui/components/Autocomplete.tsx [hunk 1]
```diff
@@ -18,6 +19,7 @@ interface IProps<T extends IBaseRestApiResponse> {
 interface IState<T> {
     fetchedItems?: Array<T>;
     loading: boolean;
+    selectedT: T | null;
```
Comment (petrjasek) [inline scripts/core/ui/components/Autocomplete.tsx:22]: what does `T` stand for in `selectedT` ?

Comment (tomaskikutis) [reply scripts/core/ui/components/Autocomplete.tsx:22]: For the generic object, same as in `Array<T>`

Comment (petrjasek) [reply scripts/core/ui/components/Autocomplete.tsx:22]: is that an idiom? if not I would go for something like `selectedItem`, looks like a typo to me

Comment (tomaskikutis) [reply scripts/core/ui/components/Autocomplete.tsx]: I don't think we use it much. I'll change it :)

**Review by petrjasek — APPROVED** [review]

**Pull request merged**